l10n_co_geo/models/res_company.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+###############################################################################
+#    License, author and contributors information in:                         #
+#    __openerp__.py file at the root folder of this module.                   #
+###############################################################################
+
+from openerp import models, fields, api
+
+
+class ResCompany(models.Model):
+    _name = 'res.company'
+    _inherit = 'res.company'
+
+    country_id = fields.Many2one(
```

Comment:
That's a design decision, if you are in Colombia, you always **need** the city, if you are dealing with an external client, you always **know** the city. Both times, they reliably give you the country. Because in colombia, the city is obligatory, we have the city `required=True`, so in the end it gives almost the same. Would you agree? Or do you have a better alternative to solve this?